### ISSUE TITLE
antlr incompatibility

### ISSUE THREAD

**Issue by abendt** [mini2Dx/gettext#5]
hi folks,

i tried to use the gettext library in one of my projects.
This project has an existing, transitive dependency on antlr. With this i couldn't successfully integrate the lib. I get errors with the message: "Could not deserialize ATN with UUID".
It seems that the underlying issue is, that gettext brings its own version of antlr, named as a different artifact (com.tunnelvisionlabs:antlr4-runtime). So we end up having different versions of antlr to be in the classpath at the same time.  I tried to use excludes/setting explicit version numbers to resolve that without success as the respective libs seem to be very sensitive to minor antlr version differences.

After some further experimentation i created a gettext fat jar that contains a shaded version of antlr. Using this i was able to use gettext in my project: https://github.com/abendt/gettext/tree/createShadowJar

would you be interested in a PR to make that change generally available?

**Comment by tomcashman** [comment]
Yep PR is welcome :-) We use `com.tunnelvisionlabs:antlr4-runtime` due to issue #1
